src/components/Header.vue

```diff
@@ -47,4 +51,37 @@ export default {
 h1 {
   width: 60%;
 }
+
 </style>
+<script>
+export default {
+  /* eslint-disable no-undef */
+  name: 'header-modal',
+  created () {
+    // This was added to get the username of the person that has logged in
+    this.$root.$on('msg', (text) => {
+      this.username = text
+    })
+  },
+  data () {
+    return {
+      username: this.$parent._data.username
+    }
+  },
+  methods: {
+    async logout_user () {
+      let that = this
+      console.log('username at logout', this.username)
+      // log the user out and reopen the login modal
+      this.$feathers.service('compositions').patch(null,
+      {
+        removeAll: this.username
+      }).then(function (result) {
+        that.$feathers.logout()
+      })
+
+      $('#loginModal').modal('show')
+    }
+  }
+}
+</script>
```

Comment:
<script> section a duplicate from lines 19 - 32

Reply:
fixed, but the comment no disappear